doc/project/checklist.md

```diff
@@ -132,6 +133,12 @@ Where appropriate, non-reset flops are used to store secure material.
 
 Shadow registers are implemented for all appropriate storage of critical control functions.
 
+### SEC_RND_CNST
+
+Compile-time random netlist constants (such as LFSR seeds or scrambling constants) are exposed to topgen via the `randtype` parameter mechanism in the comportable IP HJSON file.
+Default random seeds and permutations for LFSRs can be generated with [this script](https://github.com/lowRISC/opentitan/blob/master/hw/ip/prim/util/gen-lfsr-seed.py).
+See also related [GitHub issue](https://github.com/lowRISC/opentitan/issues/2229).
```

Comment:
Is this issue the only place where the process is documented? If so we need to add it to the documentation somewhere in canonical form.

If you really want to keep the reference to the issue:
```suggestion
See also the related [GitHub issue #2229](https://github.com/lowRISC/opentitan/issues/2229).
```

Reply:
At the moment, yes. But I agree that this needs to be documented. 
A good place for this could be the [comportable IP spec documentation](https://docs.opentitan.org/doc/rm/comportability_specification/).
We haven't really documented the parameter list feature yet in the section about the Hjson description.